students/thorn/lesson03/list_lab.py

```diff
@@ -43,6 +47,53 @@ def series1():
     return fruits
 
 def series2(fruits):
+    """ Displays the list.  Removes the last item from the list.  Redisplays 
+    the list.  Prompts the user for a fruit, removes that fruit, and display the
+    list.  
+    """
+    # fruits = list(fruits_copy)
+    print(" ".join(item for item in fruits))
+    # Using del not pop --> no need to return the item.
+    del fruits[-1]
+    print(" ".join(item for item in fruits))
+
+    # Multiple the list by 2, delete user's choice fruit.
+    fruits = fruits * 2
+    fruit_removed = input("Please select a fruit to remove: ")
```

Comment:
you could add the title call here to avoid calling title twice -> `fruit_removed = input("Please select a fruit to remove: ").title()`